openmrs-compose.yaml

```diff
@@ -0,0 +1,42 @@
+version: '2.1'
+services:
+  mysql:
+    image: mysql:5.7
+    command: "mysqld --character-set-server=utf8 --collation-server=utf8_general_ci"
+    restart: always
+    ports:
+      - "3306:3306"
+    container_name: openmrs-fhir-mysql
+    healthcheck:
+      test: "exit 0"
+    environment:
+      - MYSQL_DATABASE=openmrs
+      - MYSQL_ROOT_PASSWORD=debezium
+      - MYSQL_USER=mysqluser
+      - MYSQL_PASSWORD=mysqlpw
+      - MYSQL_ROOT_HOST=%    # Allow docker containers to connect to mysql
+    volumes:
+      - ./utils/mysql.cnf:/etc/mysql/conf.d/custom.cnf  # mysql config preconfigured to allow binlog/debezium
+      - ./data:/var/lib/mysql
+      - ./utils/dbdump:/docker-entrypoint-initdb.d # will execute any sql dumped here
```

Comment:
Sorry, I have not cloned/tried this myself yet but just from reading this, I suppose `backup.sql` is run here. How long does this take? If it is significant, maybe add a comment in the README about this too.

Reply:
it should take less than a minute on a god machine :)

Reply:
I have removed `./utils/dbdump` and `./data` to avoid these permission issues